src/Service/S3/src/Result/CopyObjectOutput.php

```diff
@@ -144,9 +144,9 @@ protected function populateResult(ResponseInterface $response, HttpClientInterfa
         $this->RequestCharged = $headers['x-amz-request-charged'][0] ?? null;
 
         $data = new \SimpleXMLElement($response->getContent(false));
-        $this->CopyObjectResult = !$data->CopyObjectResult ? null : new CopyObjectResult([
```

Comment:
spotted by test :-) base on S3 documentation.
When response contains a Payload, the there is no "intermediate" object.